CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - Note: user-defined forward declarations of any library entity has not been, and is still not, supported.
     To obtain the declaration or definition of a library entity, always include the corresponding header.
 
+### Changed
+
+- Library filenames, when compiled with MSVC (as detected by [CMake's MSVC variable](https://cmake.org/cmake/help/v3.15/variable/MSVC.html)), are now embedded with an ABI tag string, e.g. `bsoncxx-v_noabi-rhs-x64-v142-md.lib`.
```

Comment:
Consider documenting the parts of the ABI library tag in [API and ABI versioning](https://mongocxx.org/mongocxx-v3/api-abi-versioning/). That may help identify a compatibility mismatch.